quickstarts/whereami/README.md

```diff
@@ -67,20 +67,15 @@ $ export COMPUTE_REGION=#YOUR_COMPUTE_REGION# # this expects a region, not a zon
 $ export CLUSTER_NAME=whereami
 ```
 
-Now create your cluster:
+Now create your [GKE Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview) cluster:
 
 ```bash
-$ gcloud beta container clusters create $CLUSTER_NAME \
-  --enable-ip-alias \
-  --enable-stackdriver-kubernetes \
-  --region=$COMPUTE_REGION \
-  --num-nodes=1 \
-  --release-channel=regular
-
-$ gcloud container clusters get-credentials $CLUSTER_NAME --region $COMPUTE_REGION
+$ gcloud --project ${PROJECT_ID} container clusters \
+  create-auto ${CLUSTER_NAME} --region ${COMPUTE_REGION} \
+  --release-channel regular
 ```
 
-This will create a regional cluster with a single node per zone (3 nodes in total).
+This will create a regional GKE Autopilot cluster in the specified region.
```

Comment:
nit, but "regional GKE Autopilot cluster in the specified region" is redundant

Either of these shortened versions would mean the same thing:
- "regional GKE Autopilot cluster"
- "GKE Autopilot cluster in the specified region"